src/main/java/com/cloudbees/jenkins/plugins/sshagent/jna/JNRRemoteAgentStarter.java

```diff
@@ -24,15 +24,20 @@
 
 package com.cloudbees.jenkins.plugins.sshagent.jna;
 
+import jenkins.security.MasterToSlaveCallable;
+
 import com.cloudbees.jenkins.plugins.sshagent.RemoteAgent;
+
 import hudson.model.TaskListener;
-import hudson.remoting.Callable;
 import hudson.remoting.Channel;
 
 /**
  * Callable to start the remote agent.
  */
-public class JNRRemoteAgentStarter implements Callable<RemoteAgent, Throwable> {
+public class JNRRemoteAgentStarter extends MasterToSlaveCallable<RemoteAgent, Throwable> {
+
+    private static final long serialVersionUID = 5020446864184061252L;
```

Comment:
:ant: normally I would include a comment to explain that this was generated from the 1.7 version of the class and also include a `@since 1.8` javadoc comment, e.g.

```
/**
 * Ensure consistent serialization. Value generated from the 1.7 release.
 * @since 1.8
 */
```